game/scripts/npc/abilities/medusa_stone_gaze.txt

```diff
@@ -70,7 +70,7 @@
       "07"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_physical_damage"                           "40 45 50 55 60"
+        "bonus_physical_damage"                           "30 35 40 45 50" //OAA
```

Comment:
Dont nerf this. Just nerf the stone gaze talent and leave medusa unchanged outside of this.

Reply:
I didnt want to nerf this. I nerfed it because I was told to. We need another opinion on this.

Reply:
If it was just medusa's own dmg than it would be fine but this is a huge buff to any teammates you have as well. Less scaling is a must-have and Max thinks most percentages should be constant in OAA since they scale no matter what. I don't know if he gave a specific value but I think he meant like 45% all levels.

Reply:
I'm pretty sure the issue is more duration that damage amp I think the double nerf would be a rough one.

Reply:
Max has a good point. I think the best scaling for medusa's ultimate is "40 45 50 50 50" That way its the same at Dota levels without it scaling too hard at OAA levels. (Pre ultimate rework it would have been 50% at all levels).

Reply:
I did what Pocket said.